tutorials/abap-extensibility-cbo-create/abap-extensibility-cbo-create.md

```diff
@@ -2,19 +2,15 @@
 parser: v2
 auto_validation: true
 primary_tag: topic>abap-extensibility
-tags: [  tutorial>beginner, topic>abap-extensibility, topic>cloud, products>sap-s-4hana ]
+tags: [  tutorial>beginner, tutorial>license, topic>abap-extensibility, topic>cloud, products>sap-s-4hana ]
 time: 10
 ---
-
+<!--DONE in E1Y-->
 # Create a Custom Business Object
-<!-- description --> Define the data structure and create the database persistence of a custom business object.
+<!-- description -->Define the data structure and create the database persistence of a custom business object.
 
 ## Prerequisites  
-<<<<<<< HEAD
- - **Authorizations:** Your user needs a business role with business catalog **Extensibility** (ID: `SAP_CORE_BC_EXT`) in your **S/4HANA Cloud** system
-=======
  - **Authorizations:** Your user needs a business role with business catalog **Extensibility – Custom Business Objects** (ID: `SAP_CORE_BC_EXT_CBO`) in your **S/4HANA Cloud** system
->>>>>>> 59f95048a11e62962d5c8eb49e89b6f027533a25
 
 ## You will learn
 - Define a custom business object and its a simple data structure
```

Comment:
- How to define a custom business object and its (a?) simple data structure
- How to create a database table for the custom business object